torch/__future__.py

```diff
@@ -0,0 +1,15 @@
+"""
+This global flag controls whether to change the existing parameters
+in-place instead of assigning new tensors to the parameters when
+converting a module using `module._apply(fn)`.
```

Comment:
you should document the actual functions that people call (probably few know that e.g. `cuda()` calls _apply).  Just list the relevant functions.

Reply:
Fixed.